src/legacy/ui/public/vis/vis_types/_vislib_vis_legend.scss

```diff
@@ -88,6 +88,7 @@ $visLegendLineHeight: $euiSize;
 
   > * {
     max-width: 100%; // Needed for truncation (dom element has no class)
+    width: 100%;
```

Comment:
There's no need for both a `max-width` and `width` of the same value. Just remove the `max-width`.